src/helpers/xml-builder.js

```diff
@@ -1315,6 +1383,42 @@ const computeImageDimensions = (vNode, attributes) => {
   let modifiedMaxHeight;
   let modifiedMaxWidth;
 
+  // Check for HTML width and height attributes first (e.g., from TinyMCE)
```

Comment:
Question: If an image has both height,width attribute as well as style, which is preferred? I don't see a case related to this.